README.md

```diff
@@ -155,6 +155,22 @@ require "super_diff/rspec"
 
 ## Configuration
 
+If you want to use something other than the default colors, you can
+configure them by adding them to your test helper file
+(`rails_helper` or `spec_helper`):
+
+``` ruby
+SuperDiff.configure do |config|
+  config.set_insertion_color(:green)
```

Comment:
Maybe instead of `insertion` and `deletion`, what do you think about `actual` and `expected`? The reason for this is that colors are not only used in diffs but also in the initial message as well. For instance, when `eq` fails, the [first paragraph will say](https://github.com/mcmire/super_diff/blob/master/spec/integration/rspec/eq_matcher_spec.rb#L15) something like "Expected <value 1> to eq <value 2>". The first value is the "actual" value and the second is the "expected" value. Then in the [diff](https://github.com/mcmire/super_diff/blob/master/spec/integration/rspec/eq_matcher_spec.rb#L286), the removed lines come from the expected value and the added lines come from the actual value. So I think if we call these `actual_color` and `expected_color` then it will represent these two cases better.

Reply:
I reviewed some of the outputs and I think that this would be alright without being confusing.